salt/modules/rest_sample_utils.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+'''
+Utility functions for the rest_sample
+'''
+from __future__ import absolute_import
+
+__proxyenabled__ = ['rest_sample']
+
+
+def fix_outage():
+    '''
+    "Fix" the outage
+
+    Example::
```

Comment:
You had this right the first time. But I think the tests actually looks for the `CLI Example` text or something. So try this:
```
CLI Example:

.. code-block:: bash

    salt 'rest-sample-proxy' rest_sample.fix_outage
```